classes/helper/HelperView.php

```diff
@@ -53,6 +53,7 @@ public function generateView()
             'toolbar_scroll' => $this->toolbar_scroll,
             'toolbar_btn' => $this->toolbar_btn,
             'link' => $this->context->link,
+            'controller_name' => $this->controller_name,
```

Comment:
Not necessary as this variable belongs to the layout (HeaderHydrator)

Reply:
It's mandatory to let this because the content is always generated before the full template. So just putting it in the HeaderHydrator is not enough.